packages/e2e-test-utils-playwright/src/editor/get-blocks.ts

```diff
@@ -12,10 +12,19 @@ import type { Editor } from './index';
  */
 export async function getBlocks( this: Editor ) {
 	return await this.page.evaluate( () => {
+		const blocks = window.wp.data.select( 'core/block-editor' ).getBlocks();
+
 		// The editor might still contain an unmodified empty block even when it's technically "empty".
-		if ( window.wp.data.select( 'core/editor' ).isEditedPostEmpty() ) {
```

Comment:
`isEditedPostEmpty()` is always returning an empty array in the site editor for some reason 🤔 .

Reply:
I think `isEditedPostEmpty` depends on the current post, which we don't set up in Site Editor.

Reply:
Yep, but I don't think there's an alternative for the site editor, or is there? Probably doesn't need it though 🤔 ?

Reply:
There's no alternative, as far as I know. So I guess there's no need for it.